pkg/structure/structure.go

```diff
@@ -42,6 +42,17 @@ func (e EnvCondition) Check(i v1.Image) error {
 		if split[k] != v {
 			errs = append(errs, fmt.Errorf("env %q does not match %q (got %q)", k, v, split[k]))
 		}
+		if _, exists := verifyEnv[k]; exists {
+			delimiter := ":"
+			if strings.Contains(k, "LUA_") {
```

Comment:
Can you add a comment about why this is?

Or should the map be `env` -> `delimiter` ?

Reply:
Updated in [`bae3399` (#223)](https://github.com/chainguard-dev/terraform-provider-oci/pull/223/commits/bae339915d437e6d49d56f6835bfc28187d098f4). LUA uses `;` for the delimiter so I added a reference in the comments.